pygmt/clib/session.py

```diff
@@ -169,6 +170,9 @@ def info(self):
                 # "image layout": self.get_default("API_IMAGE_LAYOUT"),
                 "grid layout": self.get_default("API_GRID_LAYOUT"),
             }
+            # API_BIN_VERSION is new in GMT 6.4.0.
+            if Version(self._info["version"]) >= Version("6.4.0"):
+                self._info["binary version"] = self.get_default("API_BIN_VERSION")
```

Comment:
Is the binary version supposed to be just `6.4.0` (with no hash or date) for the official GMT 6.4 conda-forge package added in https://github.com/conda-forge/gmt-feedstock/pull/212? I was expecting a string like `6.4.0_434ed18_2022.03.11`, but am getting this output from `python -c 'import pygmt; pygmt.show_versions()'`

```
PyGMT information:
  version: v0.7.1.dev92+gde7026137
System information:
  python: 3.10.5 | packaged by conda-forge | (main, Jun 14 2022, 07:04:59) [GCC 10.3.0]
  executable: /home/username/mambaforge/envs/pygmt/bin/python
  machine: Linux-5.17.0-1016-oem-x86_64-with-glibc2.35
Dependency information:
  numpy: 1.23.2
  pandas: 1.4.3
  xarray: 2022.6.0
  netCDF4: 1.6.0
  packaging: 21.3
  geopandas: 0.11.1
  ghostscript: 9.54.0
GMT library information:
  binary version: 6.4.0
  cores: 12
  grid layout: rows
  library path: /home/username/mambaforge/envs/pygmt/lib/libgmt.so
  padding: 2
  plugin dir: /home/username/mambaforge/envs/pygmt/lib/gmt/plugins
  share dir: /home/username/mambaforge/envs/pygmt/share/gmt
  version: 6.4.0
```

and output of `mamba list | grep gmt` is:

```
dcw-gmt                   2.1.1                ha770c72_0    conda-forge
gmt                       6.4.0                h78e8b95_2    conda-forge
gshhg-gmt                 2.3.7             ha770c72_1003    conda-forge
pygmt                     0.7.1.dev92+gde7026137          pypi_0    pypi
```

Reply:
> Is the binary version supposed to be just `6.4.0` (with no hash or date) for the official GMT 6.4 conda-forge package added in [conda-forge/gmt-feedstock#212](https://github.com/conda-forge/gmt-feedstock/pull/212)? I was expecting a string like `6.4.0_434ed18_2022.03.11`.

For official releases, the version string is `6.4.0` without hash and date. `6.4.0_434ed18_2022.03.11` is the version string for dev versions.